Source/Schema.NET/Values{T1,T2,T3,T4}.cs

```diff
@@ -142,6 +142,11 @@ public Values(IEnumerable<object> items)
         /// </summary>
         public bool HasValue4 { get; }
 
+        /// <summary>
+        /// Gets whether the value of either type <typeparamref name="T1" />, type <typeparamref name="T2" />, type <typeparamref name="T3" />, or type <typeparamref name="T4" /> has a value.
+        /// </summary>
+        public bool HasValue => this.HasValue1 || this.HasValue2 || this.HasValue3 || this.HasValue4;

```

Comment:
Nice idea. This should be another PR. Looks like we might not get this one in quickly.

Reply:
Ye this was mainly so the tests would continue passing, as quite a few relied on Nullable<>.HasValue, and broke when we removed the `?`